types/node-abi/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for node-abi 2.9
```

Comment:
Should it be 2.10?

Reply:
They have the same API but I can write 2.10, yes